kolibri/core/assets/src/views/k-button/index.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+
+  <button
+    class="k-button"
+    ref="button"
+    :class="buttonClass"
+    :type="type"
+    :disabled="disabled"
+    @click="handleClick">
+    {{ text }}
+  </button>
+
+</template>
+
+
+<script>
+
+  export default {
+    props: {
+      text: {
+        type: String,
+        required: true,
+      },
+      primary: {
+        type: Boolean,
+        default: false,
+      },
+      raised: {
+        type: Boolean,
+        default: true,
+      },
+      disabled: {
+        type: Boolean,
+        default: false,
+      },
+      type: { type: String },
+    },
+    computed: {
+      buttonClass() {
+        if (this.primary && this.raised) {
+          return 'k-button-primary-raised';
+        } else if (this.primary && !this.raised) {
+          return 'k-button-primary-flat';
+        } else if (!this.primary && this.raised) {
+          return 'k-button-secondary-raised';
+        } else if (!this.primary && !this.raised) {
+          return 'k-button-secondary-flat';
+        }
+      },
+    },
+    methods: {
+      handleClick(event) {
+        this.$emit('click', event);
+        this.$refs.button.blur();
+      },
+    },
+  };
+
+</script>
+
+
+<style lang="stylus" scoped>
+
+  @require '~kolibri.styles.definitions'
+
+  $transition = all 0.4s cubic-bezier(0.25, 0.8, 0.25, 1)
+  $raised-shadow = 0 1px 5px rgba(0, 0, 0, 0.2), 0 2px 2px rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12)
+
+  .k-button
+    background: none
+    border: none
+    display: inline-block
+    cursor: pointer
+    outline: none
+    overflow: hidden
+    text-align: center
+    text-decoration: none
+    user-select: none
+    white-space: nowrap
+    margin: 8px
```

Comment:
Shouldn't have a built-in left/right margin because it causes buttons to be offset from the page's content, like this:

![image](https://user-images.githubusercontent.com/2367265/28464955-06c5a51a-6ddc-11e7-9d77-0273aa764cd2.png)


If this is to help enforce spacing between buttons, we could possibly create a simple "button group" component?